gravitee-apim-repository/gravitee-apim-repository-test/src/test/java/io/gravitee/repository/management/ApplicationRepositoryTest.java

```diff
@@ -368,22 +369,26 @@ public void shouldSearchByEnvironmentIds() throws Exception {
         );
 
         final List<Application> apps = appsPage.getContent();
-
-        assertNotNull(apps);
-        assertFalse(apps.isEmpty());
-        assertEquals(7, apps.size());
         List<String> names = apps.stream().map(Application::getName).collect(Collectors.toList());
-        assertEquals(
-            List.of(
-                APP_WITH_LONG_NAME,
-                "Application test query 1",
-                "Application test query 2",
-                "app-with-client-id",
-                "app-with-long-client-id",
-                "searched-app1",
-                "searched-app2"
-            ),
-            names
+
+        assertAll(
+            () -> assertNotNull(apps),
+            () -> assertFalse(apps.isEmpty()),
+            () -> assertEquals(7, apps.size()),
+            () ->
+                assertTrue(
+                    names.containsAll(
+                        List.of(
+                            APP_WITH_LONG_NAME,
+                            "Application test query 1",
+                            "Application test query 2",
+                            "app-with-client-id",
+                            "app-with-long-client-id",
+                            "searched-app1",
+                            "searched-app2"
+                        )
+                    )
+                )
         );
```

Comment:
I suggest to replace all assertions by one from assertj

```suggestion
        assertThat(apps)
            .map(Application::getName)
            .containsExactlyInAnyOrder(
                APP_WITH_LONG_NAME,
                "Application test query 1",
                "Application test query 2",
                "app-with-client-id",
                "app-with-long-client-id",
                "searched-app1",
                "searched-app2"
            );
```

With one assert like this in case of faillure you have a better accuracy message (be sure that you import `org.assertj.core.api.Assertions.assertThat`) and is more explicit.